### ISSUE TITLE
Use gitignore

### ISSUE THREAD

**Issue by ocombe** [Urucas/electron-packager-interactive#4]
It would be nice if the generator could use the gitignore file and not copy these ignored files into the resource folders :)

**Comment by vrunoa** [comment]
I believe this should be an implementation on the ```electron-packager``` lib, since ```electron-packager-interactive``` it just a wrapper for this module. 
Maybe it's already implemented and i'm missing something, let me check.

**Comment by ocombe** [comment]
The thing with electron packager is that you get to set the "ignore" parameter, but you cannot do that with the interactive command :(

**Comment by vrunoa** [comment]
Oh, then I'll add an implementation to the ```ignore``` param. I'll let you know when it's ready.

**Comment by ocombe** [comment]
Nice :)
I'm just afraid that it will be a bit boring to type all the files that you want to ignore every time, that's why I suggested to use the gitignore file

**Comment by vrunoa** [comment]
Inside the ```.gitignore``` is the ```node_modules``` folder which is quite important for the electron app. Maybe we should create another convention file like ```.electronignore``` to set the files to ignore or avoid to add this folder when reading the ```.gitignore```. Whats your opinion ? Wonder what @maxogden thinks ?

**Comment by ocombe** [comment]
Hmm yeah, didn't think of node_modules folder...
But if we end up making an electronignore, why not have a config file, and then why not use electron packager instead of the interactive one :)

**Comment by vrunoa** [comment]
Good point. In that case, maybe the better solution would be to create a new node cli to take all the params from from a json file. And to this module just add the implementation to accept the ```ignore``` param

**Comment by vrunoa** [comment]
hey @ocombe have you check [grunt-electron](https://github.com/sindresorhus/grunt-electron) ? With that module you can set every param as a grunt task. It's a closer approach to having a config file.

**Comment by ocombe** [comment]
Well I'm using electron-packager now, with a gulp task that I made myself :)

**Comment by vrunoa** [comment]
Cool. I will add the possibility to set the ```ignore``` param as a simple input, but it will only accept regex as ```electron-packager``` does it.
There's also a new release of the module with the icon option and automatically adding the output folder to the ignore param.
